frontend/src/common/node.ts

```diff
@@ -87,3 +87,18 @@ export const mapCorrelationToColor = (correlation: number) => {
       return '.300';
   }
 };
+
+// this is temporary until we get node type from query
+export const isSubgoal = (node: GraphNode): boolean => {
+  const { name } = node;
+  if (name.length === 3) {
+    if (name.charAt(1) !== '.') return false;
+    if (Number.isNaN(parseInt(name.charAt(0), 10))) return false;
+  } else if (name.length === 4) {
+    if (name.charAt(2) !== '.') return false;
+    if (Number.isNaN(parseInt(name.substring(0, 1), 10))) return false;
+  } else {
+    return false;
+  }
+  return true;
+};
```

Comment:
Didn't @Kapteinkrok's PR add node.type? If it did then this can be changed to use that.